components/careers/EndSection.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <section>
+    <v-stack
+      direction="column"
+      md-direction="column"
+      sm-direction="column"
```

Comment:
I guess a single direction is fine unless direction is changed for specific resolution. Can you try and check if that works?

Reply:
Good catch 👌🏼. Just `direction="column"` works for all breakpoints.

I've updated the code.